controllers/nodefeaturediscovery_controls.go

```diff
@@ -451,18 +451,24 @@ func Deployment(n NFD) (ResourceStatus, error) {
 	}
 
 	var args []string
-	port := defaultServicePort
 
-	// If the operand service port has already been defined,
-	// then set "port" to the defined port. Otherwise, it is
-	// ok to just use the defaultServicePort value
-	if n.ins.Spec.Operand.ServicePort != 0 {
-		port = n.ins.Spec.Operand.ServicePort
-	}
+	if n.ins.Spec.GrpcMode && obj.Name == nfdMasterApp {
+		// Disable NodeFeature API
+		args = append(args, "-enable-nodefeature-api=false")
```

Comment:
Should we correspondingly have `-enable-nodefeature-api=true` for the non-GrpcMode? So that also v0.13 of the operand could be used?

Reply:
🤔  next Operator release will probably be for v0.14 which comes with the API enabled by default. so... I am open to suggestions. If you think we should then I will

Reply:
I don't have a clear answer. Is the operator supposed to support some version skew? Or is it taken that the operator only works with one version of the operand 🤷

Reply:
Ah, but we have the nfd-gc as well in NFD v0.14 and possibly other breakages 🧐 Maybe the operator and operand just need to go in lock-step wrt versions

Reply:
Yup, I see the operator locked to specific NFD versions, mostly for the hard coded (Opinionated) flags and values set by the operator